scripts/config_utils.py

```diff
@@ -47,6 +47,13 @@ def preprocess_sites_config(config):
         config["process-spots"]["output_basedir"], config["core"]["batch"], "spots"
     )
 
+    # Build paths to the foci files
```

Comment:
We don't actually need this since we added `foci_dir = spot_args["output_spotdir"]` to 2.process-cells.py instead. To delete this addition do I just make another commit where I've deleted this section and add that commit to the PR?  
 
Sidenote: At least I'm not going totally crazy because I knew I had added this but somehow totally missed it when we were adding in the call to the spot_args!

Reply:
> To delete this addition do I just make another commit where I've deleted this section and add that commit to the PR?

Yep, exactly

> Sidenote: At least I'm not going totally crazy because I knew I had added this but somehow totally missed it when we were adding in the call to the spot_args!

Great detective work! 🕵️